aspnetcore/fundamentals/localization.md

```diff
@@ -314,6 +314,7 @@ Terms:
 * Culture: It is a language and, optionally, a region.
 * Neutral culture: A culture that has a specified language, but not a region. (for example "en", "es")
 * Specific culture: A culture that has a specified language and region. (for example "en-US", "en-GB", "es-CL")
+* Parent culture: The neutral culture that contains a specific culture. (for example, "en" is the parent culture of "en-US")
```

Comment:
What about this?
Parent culture: A neutral culture for one or more specific culture(s). (for example, "en" is the parent culture of "en-US", "en-GB")

Reply:
@hishamco I like it.  I did a force push to make the change without adding another commit